Felienne/hedy
# [CLEAN-UP] Improves error message when a level is not available

**Description**
In this PR we fix an issue where a generic error message was returned when a user was not allowed (or unable to) load a specific Hedy level. Instead of simply returning "No such level" we now differentiate between "Not translated" or "Not allowed by class". Making it better understandable for the user why they can't visit this level.

**Fixes**
This PR fixes #2108

**How to test**
Visit either a level that is not yet translated in your current language or restrict a level through class customizations and try to visit it. Notice that in both cases a specific error message (with correct error code either 404 or 403) is returned.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
